src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let clone = Object.assign({}, state);
+
+  return actions.map((action) => {
+    clone = Object.assign({}, clone);
+
+    switch (action.type) {
```

Comment:
Code style issue related to checklist item 3: The switch statement should have a default case for error handling.